frontend/assets/css/style.css

```diff
@@ -144,6 +152,7 @@ body {
     text-align: center;
     padding: 15px;
     padding-top: 80px;
+    border-radius: 10px;
 }
 
 .transbox p {
```

Comment:
For this, make the font size to be `15px` , then remove the font-weight, then letter spacing of `1px`